### PR TITLE
add implings

### PR DESCRIPTION
**Pull request by gc**
### Description:

-   _Write description here_

### Changes:

-   _List of changes here_

-   [] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/simulation/openables/Implings.ts [hunk 1]
```diff
@@ -0,0 +1,329 @@
+import Bank from '../../structures/Bank';
```
Comment (themrrobert) [inline src/simulation/openables/Implings.ts:1]: obviously you can see the `never used` error here

File: src/simulation/openables/Implings.ts [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+		.add('Ranarr seed', [3, 8])
+		.add('Yew seed')
+		.tertiary(50, 'Clue scroll (elite)')
+		.tertiary(128, 'Elven signet')
+});
+
+export const DragonImpling = new SimpleOpenable({
+	id: 11256,
+	name: 'Dragon impling',
+	aliases: ['dragon impling', 'dragon imp'],
```
Comment (themrrobert) [inline src/simulation/openables/Implings.ts:281]: `dimp` is a very common alias for dragon imps in game.

File: src/simulation/openables/Implings.ts [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+		.add('Dragon dart', [100, 250])
+		.add('Dragonstone', 3)
+		.add('Dragon dart tip', [100, 350])
+		.add('Dragon arrowtips', [100, 350])
+		.add('Dragon javelin heads', [25, 35])
+		.add('Babydragon bones', [100, 300])
+		.add('Dragon bones', [50, 100])
+		.add('Magic seed', 1)
+		.add('Snapdragon seed', 6)
+		.add('Summer pie', 15)
```
Comment (themrrobert) [inline src/simulation/openables/Implings.ts:301]: Dragon imp is missing the 1 in 50 elite clue drop

File: src/simulation/openables/Implings.ts [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+});
+
+export const LuckyImpling = new SimpleOpenable({
+	id: 19732,
+	name: 'Lucky impling',
+	aliases: ['lucky impling', 'lucky imp'],
```
Comment (themrrobert) [inline src/simulation/openables/Implings.ts:307]: These are also known simply as, `lucky`s if you don't have any other plans for that openable alias

File: src/simulation/openables/index.ts [hunk 5]
```diff
@@ -38,7 +52,19 @@ const openablesObject = {
 	SinisterChest,
 	BrimstoneChest,
 	LarransChest,
-	SeedPack
+	SeedPack,
+	BabyImpling,
```
Comment (themrrobert) [inline src/simulation/openables/index.ts:56]: Wouldn't this be cleaner:
```ts
import { Implings } from './Implings'
...
const openablesObject = {
...
  SeedPack,
  ...Implings
}
```

File: src/simulation/openables/index.ts [hunk 6]
```diff
@@ -7,6 +7,20 @@ import ElvenCrystalChest from './ElvenCrystalChest';
...
+	EssenceImpling,
+	GourmetImpling,
+	LuckyImpling,
+	MagpieImpling,
+	NatureImpling,
+	NinjaImpling,
+	YoungImpling
+} from './Implings';
 import LarransChest from './LarransChest';
 import LuckyImp from './LuckyImp';
```
Comment (themrrobert) [inline src/simulation/openables/index.ts:25]: Remove the old "LuckyImp" import, + and entry from `const openablesObject`

File: src/simulation/openables/Implings.ts [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+		.oneIn(100, 'Adamant kiteshield')
+		.oneIn(100, "Blue d'hide chaps")
+		.oneIn(100, 'Red spiky vambraces')
+		.oneIn(100, 'Rune dagger')
+		.oneIn(100, 'Battlestaff')
+		.oneIn(100, 'Adamantite ore', 10)
+		.oneIn(100, "Slayer's respite", 2)
+		.oneIn(100, 'Wild pie')
+		.oneIn(100, 'Watermelon seed', 3)
+		.oneIn(100, 'Diamond')
```
Comment (themrrobert) [inline src/simulation/openables/Implings.ts:167]: Eclectic is missing 1 in 25 Medium clue

**Review by themrrobert — CHANGES_REQUESTED** [review]

File: src/simulation/openables/index.ts (hunk 5 above)
Comment (gc) [reply src/simulation/openables/index.ts:56]: I wanted them as an array, and you cant spread an array over an object like that, not a big deal

Comment (themrrobert) [reply src/simulation/openables/index.ts:56]: ah, yep, makes sense

**Pull request merged**